SearchProviders/vespa.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Vespa",
+  "active": true,
+  "default": false,
+  "connector": "VespaRestSearch",
+  "url": "https://localhost:8080/search/",
+  "query_template": "{url}?query={query_string}&fields={fields}&hits={hits}",
+  "query_processors": ["AdaptiveQueryProcessor"],
+  "query_mappings": "",
+  "result_processors": [
+    "MappingResultProcessor",
+    "LenLimitingResultProcessor",
+    "CosineRelevancyResultProcessor"
+  ],
+  "response_mappings": "RESULTS=root_object.hits.hits, #FOUND=root_object.hits.found, #RETRIEVED=root_object.hits.retrieved",
```

Comment:
Sorry @HarshMN2345 please remove the hash marks (#) it should be 
FOUND=root_object.hits.found, not #FOUND=...